src/test/java/com/puppycrawl/tools/checkstyle/meta/JavadocMetadataScraperTest.java

```diff
@@ -192,6 +193,21 @@ public void testWriteTagCheck() throws Exception {
                 "expected correct parse");
     }
 
+    @Test
+    public void testViolationsEmptyDescriptionModule() throws Exception {
```

Comment:
We need a new test for `JavadocMetadataScrapper` for the logging of the violation. I am not seeing any changes to it.

Reply:
To be done in separate PR-
> We need a new test for JavadocMetadataScrapper for the logging of the violation

You want to check the the exact violations?

Reply:
Created one now.